src/notes/StructuredFieldPlugin.jsx

```diff
@@ -691,7 +688,8 @@ function createStructuredField(opts, shortcut) {
     let nodes = [Slate.Text.createFromString(String(shortcut.getText()))];
     const isInserter = shortcut instanceof InsertValue;
     if (isInserter) {
-        const lines = String(shortcut.getText()).split(/\n\r|\r\n|\r|\n/g);
+        let lines;
+        lines = String(shortcut.getDisplayText()).split(/\n\r|\r\n|\r|\n/g);
```

Comment:
[MG3] Tiny, but there's no need for 2 lines here or a `let`. It can just be `const lines = ...`